README.md

```diff
@@ -1 +1,186 @@
+
+## Implementation Requirements
+
+When a user runs this program, the program should politely ask the user for the following things:
+
+1. a word to combine to begin the portmanteau
+1. a second word to combine and begin the portmanteau
+
+Then, the program should use the two input words in to output and display the result of the following rules and create one portmanteau.
+
+Here is an example of what your program could look like. It is not required to look exactly like this:
+![example screenshot of a calculator proram](example.png)
+
+### Wave 1
+
+In this wave, work on the following features:
+- establishing the `run_generator` method
+- getting all of the input needed
+- storing those pieces of input into variables
+- being able to print out those variables
+- iterating through individual characters in words to check which characters are vowels
+
+#### `run_generator` method
+
+Make a method named `run_generator`. This method has no parameters. Start this method by outputting a prompt to ask for a first input word A, and accept some input. Then, ask for a second input word B, and accept some input.
+
+Make sure you save these inputs into variables. Then, output these variables by printing them to the console, and the statement `"in the run_generator method"`.
+
+Then, _call_ or _invoke_ the `run_generator` outside of the method definition, at the bottom of the `generator.rb` file.
+
+Confirm that this is all setup by running `$ ruby generator.rb`, and make sure you can input two words, and you eventually see the statement `"in the run_generator method"` in the console.
+
+#### `is_vowel?` method
+
+Next, make a method named `is_vowel?`. This method should:
+- take in one argument, a string, which will represent one letter. Name this argument `letter`
+- return either `true` or `false`-- true if it is either the letters `a`, `e`, `i`, `o`, or `u`, false if it is anything else
+
+Test that this method works by calling it inside of the `run_generator` method, and passing in some test cases, and printing the result to the console. Here is an example of what that might look like:
+
+```ruby
+def run_generator
+  # ... maybe some other code ...
+
+  test_vowel = "i"
+  puts "Is #{test_vowel} a vowel?"
+  puts is_vowel?(test_vowel)
+
+  test_consonant = "g"
+  puts "Is #{test_consonant} a vowel?"
+  puts is_vowel?(test_consonant)  ...
+
+  # ... maybe some other code ...
+end
+```
+
+### Wave 2
+
+In wave two, work on combining the first two input words like portmanteau rules.
+
+Implement these rules:
+
+#### Input validation rules
+
+- The input for the first word must be at least two characters. If the input is less than 2 characters (as in 0 or 1 characters), the program should handle it in the following method:
+  - output a message to the command line that informs the user that the input was invalid and must be at least 2 characters
+  - asks the user to re-enter the input for the same prompt
```

Comment:
Line 99: "method" is a keyword, did you intend to use it that way?

Reply:
Is this more difficult than the error checking we had them do in Calculator?

Reply:
@droberts-ada wrt the question about error checking: Ideally, this means:

```ruby
input = gets.chomp
until ( input.length > 1 )
  puts "the input was invalid and must be at least 2 characters"
  input = gets.chomp
end
```

What do you think?